sonar/modules/users/jsonschemas/users/user-v1.0.0.json

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://sonar.ch/schemas/users/user-v1.0.0.json",
+  "additionalProperties": false,
+  "title": "SONAR user v1.0.0",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "default": "https://sonar.ch/schemas/users/user-v1.0.0.json"
+    },
+    "pid": {
+      "title": "User identifier",
+      "description": "User persistent identifier.",
+      "type": "string"
+    },
+    "user_id": {
+      "title": "User account identifier",
+      "description": "User account identifier.",
+      "type": "integer"
+    },
+    "first_name": {
+      "title": "First name",
+      "validationMessage": "First Name is required.",
+      "type": "string",
+      "minLength": 2
+    },
+    "last_name": {
+      "title": "Last name",
+      "validationMessage": "Last Name is required.",
```

Comment:
validationMessage is not a json schema properties as I know. This should be in the form option. (Same for the entire file.)

Reply:
Ok, I'll remove these entries. For information, validationMessage entry exists in json schemas in RERO ILS, too : https://github.com/rero/rero-ils/search?p=1&q=validationMessage&unscoped_q=validationMessage

Reply:
Can you check with the rero-ils team? Thanks.